### ISSUE TITLE
Applying local policy goes into a continuous loop

### ISSUE THREAD

**Issue by Blason** [openappsec/openappsec#77]
Hi Team,

I am trying to create a local policy and have attached my policy here. However when I apply the policy it goes on and on.
Any idea how do I debug or check what are errors in file?

`open-appsec-ctl
[local_policy.yaml.txt](https://github.com/openappsec/openappsec/files/13390005/local_policy.yaml.txt)
-ap local_policy.yaml
Applying new policy. Policy path: local_policy.yaml
.............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................^C
`

**Comment by orianelou** [comment]
Hi @Blason, 

Thank you for attaching the policy! We have successfully recreated the issue and are looking into it. I will update yo once we find a solution.

**Comment by Blason** [comment]
Thanks man!! That means the policy is properly created,

**Comment by Blason** [comment]
Hey folks any luck with the policy?

**Comment by orianelou** [comment]
Hi @Blason, 

We're still looking into a fix, but to get you up and running you can try this workaround to reload the policy:
```
cpnano -qs orchestration 
cpnano -rs orchestration
```

**Comment by Blason** [comment]
I see - that is a good workaround

**Comment by SpiderD555** [comment]
Just a small note about the bug. I think I have also encountered the same bug - in my case the problem was with wrong yaml policy formatting. Once corrected it all started to work correctly again.

**Comment by bmbeverst** [comment]
I think I have run into the same issue. I also had a continuous loop even after I corrected the policy. I copied the correct policy for a working node.

@Blason does your open-appsec pod have any values in the `/etc/cp/conf/policy.json` file? Furthermore, how about the output of `/usr/sbin/open-appsec-ctl -s`?

Run commands on the open-appsec pod with this command, `kubectl exec -it <YOUR_OPEN_APPSEC_K8S_NGINX_INGRESS_POD_NAME> -- sh`
